src/data/history.ts

```diff
@@ -230,6 +230,10 @@ export const computeHistory = (
       unit = hass.config.unit_system.temperature;
     } else if (computeStateDomain(stateInfo[0]) === "humidifier") {
       unit = "%";
+    } else if (computeStateDomain(stateInfo[0]) === "counter") {
+      unit = "#";
+    } else if (computeStateDomain(stateInfo[0]) === "input_number") {
```

Comment:
We should probably also add `number` here?

Reply:
I wasn't aware of the `number`integration :see_no_evil: it is now added as well